src/serde_jcs/mod.rs

```diff
@@ -0,0 +1,5 @@
+mod entry;
+mod serializer;
+
+pub use self::serializer::to_string;
+pub use self::serializer::to_vec;
```

Comment:
Consider making whole `serde_jcs` a separate lib crate in this repo as it cargo shows a warning that some parts are not used. Or mark `#[allow(dead_code)]` temporarily